.gitignore

```diff
@@ -64,5 +64,5 @@ coverage
 # Artifacts
 /tools/commit_slider/*.json
 /tools/commit_slider/slider_cache/*
-/src/plugins/intel_cpu/thirdparty/ComputeLibrary/build/*
```

Comment:
Why this line has to be changed?

Reply:
@praasz Since the request concerns not only refactoring the ACL configuration but also moving the ACL build process from source directories to our main build directory, ignoring the build directory is no longer necessary. The arm_compute_version.embed file remains, as it is required by the oneDNN dependency.